### PR TITLE
Abb161 correct arabic works

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
It looks like this PR went wrong, created another one.

### CODE REVIEW

**Comment by eu-genia** [comment]
6914 was already merged here https://github.com/BetaMasaheft/Works/pull/1044 and should be now deleted in this PR, not updated

File: 2001-3000/LIT2077Nuzhat.xml [hunk 1]
```diff
@@ -4,46 +4,69 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
             </availability>
          </publicationStmt>
-         <sourceDesc>
-            <p>Work of the literatures of Ethiopia and Eritrea</p>
-            <p>A TEI file converted from the ETHIO-authority google spreadsheet</p>
+         <sourceDesc>     
+               <listWit>
+                  <witness corresp="AMS00004" xml:id="L">Agaro / Aggārō, Agaro Muḥammad Sayf, AMS00004;</witness>
+                  <witness corresp="MAJ00060" xml:id="P">Jimata, Muḥammad Abba Jamaal, MAJ00060</witness>                
+               </listWit>
```
Comment (eu-genia) [inline 2001-3000/LIT2077Nuzhat.xml:28]: and also BNFabb161 to complete the list

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6914NuzhatAsrar.xml [hunk 2]
```diff
@@ -6,7 +6,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <fileDesc>
             <titleStmt>
```
Comment (eu-genia) [inline new/LIT6914NuzhatAsrar.xml:7]: please delete this file and commit the change to this branch and PR

File: 2001-3000/LIT2077Nuzhat.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply 2001-3000/LIT2077Nuzhat.xml:29]: done

File: new/LIT6914NuzhatAsrar.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LIT6914NuzhatAsrar.xml:7]: done

Comment (eu-genia) [reply new/LIT6914NuzhatAsrar.xml:7]: it is not deleted. you must delete 6914

**Comment by DenisNosnitsin1970** [comment]
I thoufgt I have deleted it, and it looked so, 
`https://github.com/BetaMasaheft/Works/pull/1045/commits/73ace359b2cb6aa8410b4c120892a50b7d7cb08e`. What if you let PR though, and if 6914 is still there, I will delete it in a separate round? (Before I though it should be the procedure).

**Comment by eu-genia** [comment]
In this round please, it had already been merged, no need to sync, please delete in this branch

**Comment by eu-genia** [comment]
It seems that you deleted it but then you got out of sync and remerged the branch 
if you look in the github desktop history you will see that you recreated it by remerging
This is why it is very important to always sync by fetch and pull in github desktop before committing any changes

**Comment by eu-genia** [comment]
I have now reverted the latest commit which was the restoration of the record to be deleted by the merging of the unsynced branch

**Review by eu-genia — APPROVED** [review]
I have fixed the issue please always sync your branches to avoid overwriting your own changes by remerging from the unsynced version

**Comment by DenisNosnitsin1970** [comment]
Thank you! :)

**Pull request merged**